Neuron/Sections/Transaction/Send/SendTransactionViewController.swift

```diff
@@ -113,7 +121,7 @@ class SendTransactionViewController: UITableViewController, TransactonSender {
             let realm = try! Realm()
             let ether = realm.objects(TokenModel.self).first(where: { $0.type == .ether })!
             if ether.tokenBalance < paramBuilder.txFeeNatural {
-                Toast.showToast(text: "请确保账户剩余\(token.gasSymbol)高于矿工费用，以便顺利完成转账～")
+                Toast.showToast(text: "Transaction.Send.balanceNotSufficient".localized().replacingOccurrences(of: "[symbol]", with: token.gasSymbol))
```

Comment:
Also since this string construction is used and duplicated a few times I'd suggest we extract a private func for it.